### PR TITLE
Add Shields to README

### PR DESCRIPTION
**Pull request by roryjamesallen**
Groundwork for adding shields to the README.
I've done the easy ones (socials), next is the suggestions from the Discord, so version, pydocs, contributor count.

### CODE REVIEW

**Comment by awonak** [comment]
Here are some suggestions, pick any that look useful or interesting to you:


GitHub release (latest SemVer)
```
![GitHub release (latest SemVer)](https://img.shields.io/github/v/release/Allen-Synthesis/Europi)
```

GitHub contributors
```
![GitHub contributors](https://img.shields.io/github/contributors/Allen-Synthesis/EuroPi)
```

GitHub repo file count (file extension)
```
![GitHub repo file count (file extension)](https://img.shields.io/github/directory-file-count/Allen-Synthesis/EuroPi/software/contrib?extension=py&label=contrib%20scripts&type=file)
```

GitHub all releases
```
![GitHub all releases](https://img.shields.io/github/downloads/Allen-Synthesis/EuroPi/total?label=release%20downloads)
```

PyPI - Downloads
```
![PyPI - Downloads](https://img.shields.io/pypi/dm/micropython-europi?label=PyPi%20downloads)
```

Search around https://shields.io/ if you're curious about other options.

**Comment by awonak** [comment]
Also, here's some alternative MarkDown versions of the social tags:

Subreddit subscribers
```
![Subreddit subscribers](https://img.shields.io/reddit/subreddit-subscribers/EuroPi?style=social)
```

Subreddit subscribers
```
![Subreddit subscribers](https://img.shields.io/reddit/subreddit-subscribers/EuroPi?style=flat)
```

Discord
```
![Discord](https://img.shields.io/discord/931297838804127794?style=social)
```


Discord
```
![Discord](https://img.shields.io/discord/931297838804127794?label=Discord&style=flat)
```

File: README.md [hunk 1]
```diff
@@ -1,6 +1,27 @@
...
+<p>
+  <a href="https://discord.gg/JaQwtCnBV5">
+        <img src="https://discordapp.com/api/guilds/931297838804127794/widget.png?style=shield"
+            alt="chat on Discord">
+  </a>
+  <a href="https://www.reddit.com/r/EuroPi/">
+        <img src="https://img.shields.io/reddit/subreddit-subscribers/europi?style=social"
+            alt="chat on Discord">
+  </a>
+  <a href="">
```
Comment (awonak) [inline README.md:17]:
```suggestion
  <a href="https://github.com/Allen-Synthesis/EuroPi/tree/main/software/contrib">
```

File: README.md [hunk 2]
```diff
@@ -1,6 +1,27 @@
 # EuroPi
 
-The EuroPi is a fully user reprogrammable module based on the [Raspberry Pi Pico](https://www.raspberrypi.com/products/raspberry-pi-pico/), which allows users to process inputs and controls to produce outputs based on code written in Python. The entire project is open-source.
+
+### The EuroPi is a fully user reprogrammable module based on the [Raspberry Pi Pico](https://www.raspberrypi.com/products/raspberry-pi-pico/), which allows users to process inputs and controls to produce outputs based on code written in Python. The entire project is open-source.
```
Comment (awonak) [inline README.md:4]: [optional] I think the larger font here is a little distracting, imho it's fine without.

File: README.md [hunk 3]
```diff
@@ -1,6 +1,27 @@
...
+
+---
+
+<p>
```
Comment (awonak) [inline README.md:8]: [optional] Most repos will start with the main title, immediately followed by the shields, then the summary paragraph. It's fine in this order too though if you prefer.

**Review by awonak — APPROVED** [review]

**Review by mjaskula — APPROVED** [review]
Looks good to me. I'd like to see some more developer-focused metrics here as well, since this is github. For example, the release version as @awonak suggested, or the latest build status. But this is a good start.

**Pull request merged**